cogs/currency.py

```diff
@@ -112,6 +114,15 @@ async def create_bank_transaction(self, c, user: discord.Member,
             "INSERT INTO BankTransactions(UserID, Amount, Action, "
             "Metadata, Date) VALUES(?, ?, ?, ?, ?)", t)
 
+    async def give_user_cheeps(self, ctx, amount: Decimal, action: str,
```

Comment:
the bot doesnt have an explicit name for currency
also this function isnt needed, use `create_bank_transaction`